watson/utils.py

```diff
@@ -56,36 +65,29 @@ def confirm_tags(tags, watson_tags):
     return True
 
 
-def style(name, element):
-    def _style_tags(tags):
-        if not tags:
-            return ''
+def format_short_id(id):
+    return style('id', id[:7])
```

Comment:
I'm not sure to understand why we need those functions. I find the old behavior less intrusive, and I think it can be compatible with this feature.